context/http.go

```diff
@@ -175,6 +175,19 @@ func GetResponseLogger(ctx Context) Logger {
 	return l
 }
 
+func GetErrorResponseLogger(ctx Context, errorMap map[string]interface{}) Logger {
```

Comment:
I guess I just don't think this function signature is at the level of the rest of this package. It requires a lot of work for the caller and doesn't hint at proper usage. It can be replaced with the following (with some changes to the `Logger` interface):

```go
GetResponseLogger(ctx).WithFields({"err.code": err.Code, ...})
```